src/pages/vrf/v2/subscription/examples/test-locally.mdx

```diff
@@ -0,0 +1,160 @@
+  Even though local testing has several benefits, testing with a VRF mock covers the bare minimum of use cases. Make
+  sure to test your consumer contract throughly on public testnets.
+</Aside>
+
+## Benefits of local testing
+
+<ContentCommon section="localTestingBenefits" />
+
+## Testing logic
+
+Complete the following tasks to test your VRF v2 consumer locally:
+
+1. Deploy the [VRFCoordinatorV2Mock](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol). This contract is a mock of the [VRFCoordinatorV2](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/VRFCoordinatorV2.sol) contract.
+1. Call the VRFCoordinatorV2Mock [createSubscription function](https://github.com/smartcontractkit/chainlink/blob/abf1d19c147c4fd54173b48140706780b53be474/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol#L176) to create a new subscription.
+1. Call the VRFCoordinatorV2Mock [fundSubscription function](https://github.com/smartcontractkit/chainlink/blob/abf1d19c147c4fd54173b48140706780b53be474/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol#L138) to fund your newly created subscription. **Note**: You can fund with an arbitrary amount.
+1. Deploy your VRF consumer contract.
+1. Call the the VRFCoordinatorV2Mock [addConsumer function](https://github.com/smartcontractkit/chainlink/blob/abf1d19c147c4fd54173b48140706780b53be474/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol#L229) to add your consumer contract to your subscription.
+1. Request random words from your consumer contract.
+1. Call the VRFCoordinatorV2Mock [fulfilRandomWords function](https://github.com/smartcontractkit/chainlink/blob/abf1d19c147c4fd54173b48140706780b53be474/contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol#L88) to fulfill your consumer contract request.
+
+## Testing
+
+### Open the contracts on RemixIDE
+
+Open _VRFCoordinatorV2Mock_ and compile in Remix:
+
+<CodeSample src="samples/VRF/mock/VRFCoordinatorV2Mock.sol" />
+
+Open _VRFv2Consumer_ and compile in Remix:
+
+<CodeSample src="samples/VRF/mock/VRFv2Consumer.sol" />
+
+Your RemixIDE file explorer should display _VRFCoordinatorV2Mock.sol_ and _VRFv2Consumer.sol_:
+
+<ClickToZoom src="/images/vrf/mock/v2-subscription-remix-fileexplorer.jpg" />
+
+### Deploy VRFCoordinatorV2Mock
+
+1. Open _VRFCoordinatorV2Mock.sol_.
+
+1. Under _DEPLOY & RUN TRANSACTIONS_, select _VRFCoordinatorV2Mock_.
+
+   <ClickToZoom src="/images/vrf/mock/v2-deploymock.jpg" />
+
+1. Under _DEPLOY_, fill in the `_BASEFEE` and `_GASPRICELINK`. These variables are used in the _VRFCoordinatorV2Mock_ contract to represent the base fee and the gas price (in LINK tokens) for the VRF requests. You can set: `_BASEFEE=100000000000000000` and `_GASPRICELINK=1000000000`.
+
+1. Click on _transact_ to deploy the _VRFCoordinatorV2Mock_ contract.
+
+1. Once deployed, you should see the _VRFCoordinatorV2Mock_ contract under _Deployed Contacts_.
```

Comment:
```suggestion
1. Once deployed, you should see the _VRFCoordinatorV2Mock_ contract under _Deployed Contracts_.
```

Reply:
Search for additional instances in this file and also apply changes to direct funding file